testData/solutions/win32.json5

```diff
@@ -831,63 +1037,86 @@
+                            "default": 0
                         }
                     },
-                    "Options.AllowMouseEchoWhenMuted": {
+                    "Options.VirtualDocumentLinkActivationMethod": {
                         "schema": {
-                            "title": "Allow Mouse Echo",
-                            "description": "Whether to echo mouse movement when otherwise muted.",
-                            "enum": [
-                                0,
-                                1
-                            ],
-                            "enumLabels": [
-                                "off",
-                                "on"
-                            ],
+                            "title": "Link Activation",
+                            "description": "How to activate a link when the enter key is pressed. Either simulates a mouse click or passes along the enter keypress.",
+                            "enum": [0,1],
+                            "enumLabels": ["Enter simulates mouse click.", "Enter sends Enter key."],
                             "default": 0
                         }
                     },
-                    "Options.AllowSpeechOnDemandWhenMuted": {
+                    "HTML.ButtonTextOptions": {
                         "schema": {
-                            "title": "Allow Reading Commands",
-                            "description": "Whether to allow speech on demand when otherwise muted.",
+                            "title": "Button Text Options",
+                            "description": "Indicate which elements from a Button speak.",
                             "enum": [
                                 0,
-                                1
+                                1,
+                                2,
+                                3,
+                                4,
+                                5
                             ],
                             "enumLabels": [
-                                "off",
-                                "on"
+                                "Title",
+                                "Screen Text",
+                                "Alt",
+                                "Value",
+                                "Longest of Above",
+                                "The first attribute found in the order list"
                             ],
                             "default": 1
                         }
                     },
-                    "Options.AllowTypingEchoWhenMuted": {
+                    "HTML.CustomButtonAttrs": {
+                        "schema": {
+                            "title": "Custom Button Attributes",
+                            "description": "Specify the order in which attributes of Buttons should be searched.",
+                            "type": "string",
+                            "default": "text|title|alt|href"
```

Comment:
Another good candidate for an intra-application transform (aka "alias").

Reply:
this has been commented out with a comment.